### PR TITLE
fix(chat): fix import export (issues: #640, #768, #767) 

### PR DESCRIPTION
**Pull request by denys-kolomiitsev**
Issues:

- #640 
- #768 
- #767

### CODE REVIEW

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-802.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-802.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat/src/store/import-export/importExport.epics.ts [hunk 1]
```diff
@@ -241,27 +248,29 @@ const importConversationsEpic: AppEpic = (action$) =>
...
       ).pipe(
+        toArray(),
         switchMap(() => {
           return ConversationService.getConversations(undefined, true).pipe(
-            catchError((err) => {
-              console.error(
-                'An error occurred while uploading conversations and folders:',
-                err,
+            catchError(() => {
+              toast.error(
```
Comment (mikitabut) [inline apps/chat/src/store/import-export/importExport.epics.ts]: use action for showing error instead

File: apps/chat/src/store/import-export/importExport.epics.ts [hunk 2]
```diff
@@ -330,6 +346,7 @@ const importZipEpic: AppEpic = (action$, state$) =>
                 toast.error(errorsMessages.unsupportedDataFormat);
```
Comment (mikitabut) [inline apps/chat/src/store/import-export/importExport.epics.ts]: Recheck all toast.error and use actions instead

File: apps/chat/src/store/import-export/importExport.epics.ts [hunk 3]
```diff
@@ -392,75 +409,105 @@ const uploadConversationAttachmentsEpic: AppEpic = (action$) =>
...
+                      name,
+                      absolutePath,
+                      relativePath,
+                      status,
+                      percent,
+                      contentType,
+                      oldId: attachment.id,
+                    },
+                  });
+                }
```
Comment (Alexander-Kezik) [inline apps/chat/src/store/import-export/importExport.epics.ts]:
```suggestion
              map(({ percent, result }) => {
                if (result) {
                  return ImportExportActions.uploadSingleAttachmentSuccess({
                    apiResult: {
                      ...result,
                      oldId: attachment.id,
                    },
                  });
                }
```

Comment (denys-kolomiitsev) [reply apps/chat/src/store/import-export/importExport.epics.ts]: Thank you for the suggestion!
Redundant object destruction removed.

File: apps/chat/src/store/import-export/importExport.epics.ts (hunk 1 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/store/import-export/importExport.epics.ts]: Thank you for the suggestion!
Changed to use 'UIActions.showErrorToast'

File: apps/chat/src/store/import-export/importExport.epics.ts (hunk 2 above)
Comment (denys-kolomiitsev) [reply apps/chat/src/store/import-export/importExport.epics.ts]: Thank you for the suggestion!
Changed to use 'UIActions.showErrorToast' in all cases.

**Comment by denys-kolomiitsev** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-802.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**